assets/css/components/rich-text-renderer.scss

```diff
@@ -0,0 +1,3 @@
+.rich-text-renderer {
+    @apply mt-5 pb-10;
```

Comment:
je gebruikt mt-5 maar op de titel staat al en padding bottom. nu is de spacing tussen de titel tekst te groot. mt-5 zou dus weg kunnen

Reply:
de pb-10 is te groot en klopt volgens design niet. het zou hier pb-5 moeten zijn